recipes/av2/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "av2" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/argoverse2-api/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 853aca3c9dd561f189b6c808f3d73ea816fc5e2cce7d98fc3c832dd181c4ef45
+
+build:
+  noarch: python
+  skip: true
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - setuptools >=42
+    - wheel
+
+  host:
+    - numpy
+    - pip
+    - python >=3.8
+
+  run:
+    - av
+    - click
+    - joblib
+    - matplotlib-base
+    - nox
+    - numba
+    - numpy >=1.21.5
+    - opencv
+    - pandas
+    - python >=3.8
+    - pip
+    - pyarrow
+    - rich
+    - scipy
+
+test:
+  imports:
+    - av2
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/argoai/argoverse2-api
+  summary: 'The official API of the Argoverse 2 dataset.'
+  description: |
+    The official API of the Argoverse 2 dataset.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

```suggestion
  license_file: 
    - LICENSE
    - NOTICE
```
The notice probably also needs to be included.